### PR TITLE
feat: Staking limit alerts + update LidoDAO ABI

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: ethereum-operations/src/agent-dao-ops.ts [hunk 1]
```diff
@@ -205,6 +212,59 @@ async function handleDepositExecutorBalance(
...
+  const lidoDao = new ethers.Contract(
+    LIDO_DAO_ADDRESS,
+    LIDO_DAO_ABI,
+    ethersProvider
+  );
+  const stakingLimitInfo = await lidoDao.functions.getStakeLimitFullInfo({
+    blockTag: blockEvent.block.number,
+  });
+  const currentStakingLimit = new BigNumber(
+    String(stakingLimitInfo.currentStakeLimit)
```
Comment (skhomuti) [inline ethereum-operations/src/agent-dao-ops.ts:226]: why cast to string? isn't that a bignumber already?

File: ethereum-operations/src/agent-dao-ops.ts [hunk 2]
```diff
@@ -205,6 +212,59 @@ async function handleDepositExecutorBalance(
...
+  } else if (
+    lastReportedStakingLimit30 + REPORT_WINDOW_STAKING_LIMIT_30 < now &&
+    currentStakingLimit.isLessThan(maxStakingLimit.times(0.3))
+  ) {
+    findings.push(
+      Finding.fromObject({
+        name: "Staking limit below 30%",
+        description:
+          `Current staking limit is ${currentStakingLimit.toFixed(2)} ETH ` +
+          `this is lower than 10% of max staking limit ` +
```
Comment (skhomuti) [inline ethereum-operations/src/agent-dao-ops.ts:257]: 30% typo

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: ethereum-operations/src/agent-dao-ops.ts (hunk 1 above)
Comment (dgusakov) [reply ethereum-operations/src/agent-dao-ops.ts:226]: There are several BigNumbers. One from ethers does not fit our needs

File: ethereum-operations/src/agent-dao-ops.ts (hunk 2 above)
Comment (dgusakov) [reply ethereum-operations/src/agent-dao-ops.ts]: fixed

**Review by skhomuti — APPROVED** [review]

**Pull request merged**